src/main/java/graphql/execution/batched/DataFetchingException.java

```diff
@@ -0,0 +1,26 @@
+package graphql.execution.batched;
```

Comment:
ExceptionWhileDataFetching is not a throwable/exception. DataFetchingException shouldn't be specific to BatchedExecutionStrategy, I see more usecases for it.